nifi-ngsi-bundle/nifi-ngsi-processors/src/main/java/org/apache/nifi/processors/ngsi/ngsi/utils/NGSIUtils.java

```diff
@@ -150,7 +150,7 @@ private AttributesLD parseNgsiLdAttribute(String key, JSONObject value) {
         } else if ("Property".contentEquals(attrType)) {
             attrValue = value.get("value");
         } else if ("GeoProperty".contentEquals(attrType)) {
-            attrValue = value.getJSONObject("value").get("coordinates").toString();
+            attrValue = value.getJSONObject("value").getString("type");
```

Comment:
You are no longer getting the value of the attribute in the `attrValue` variable...